### PR TITLE
Add docs on JinaAI embedding module

### PR DESCRIPTION
**Pull request by JoanFM**
Hey Weaviate team!

We just launched [Jina Embeddings](https://jina.ai/embeddings/) and would love to add Documentation on the module that would allow users to use them as a module in Weaviate.

Thanks!

PR to Weaviate: https://github.com/weaviate/weaviate/pull/3725

### CODE REVIEW

**Comment by JoanFM** [comment]
> To avoid any confusion in the future about your contribution to Weaviate, we work with a [Contributor License Agreement](https://weaviate.io/service/contributor-license-agreement). If you agree, you can simply add a comment to this PR that you agree with the CLA so that we can merge.beep boop - the Weaviate bot 👋🤖PS:Are you already a member of the [Weaviate Slack channel?](https://weaviate.io/slack)

I agree

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+## Overview
+
+Weaviate uses the `text2vec-jinaai` module to obtain vectors.
+- [JinaAI Embeddings](https://jina.ai/embeddings/)
+
+Key notes:
+
+- As it uses a third-party API, you will need an API key.
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:15]: Consider:
JinaAI requires a third-party API key.

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- Enabling this module will enable the [`nearText` search operator](/developers/weaviate/api/graphql/search-operators.md#neartext).
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:16]: Consider:
To use the [`nearText` search operator], enable the `text2vec-jinaai` model.

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+- The default model is `jina-embeddings-v2-base-en`.
+
+### Docker Compose file
+
+To use `text2vec-jinaai`, you must enable it in your Docker Compose file (`docker-compose.yml`). You can do so manually, or create one using the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator).
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:21]: Consider:
You can edit the Docker Compose file manually, or use the the [Weaviate configuration tool](/developers/weaviate/installation/docker-compose.md#configurator) to create a custom file.

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+#### Parameters
+
+|Parameter|Required|Purpose|
+|:-|:-|:-|
+|`ENABLE_MODULES`|Required|The modules to enable. Include `text2vec-jinaai` to enable the module.|
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:27]: Consider 
|`ENABLE_MODULES`|Yes|

Are there other models? We should either list them or link to a list

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+|`DEFAULT_VECTORIZER_MODULE|Optional|The default vectorizer module. You can set this to `text2vec-jinaai` to make it the default for all classes.|
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:28]: Consider:
|`DEFAULT_VECTORIZER_MODULE|No|


Consider:
To make `text2vec-jinaai` the default for all classes, set it here.

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+|`JINAAI_APIKEY`|Optional|Your JinaAI API key. You can also provide the key at query time.|
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:29]: Consider:
|`JINAAI_APIKEY`|No|

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+#### Example
+
+This configuration enables `text2vec-jinaai`, sets it as the default vectorizer, and sets the API keys.
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:33]: Consider.

```
This Docker Compose file configures JinaAI.
- It enables `text2vec-jinaai`.
- It sets `text2vec-jinaai` as the default vectorizer.
- It sets the JinaAI keys.
```

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+      # highlight-start
+      ENABLE_MODULES: text2vec-jinaai
+      DEFAULT_VECTORIZER_MODULE: text2vec-jinaai
+      JINAAI_APIKEY: sk-foobar  # Setting this parameter is optional; you can also provide the key at query time.
+      # highlight-end
+      CLUSTER_HOSTNAME: 'node1'
+...
+```
+
+## Class configuration
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:57]: Consider

```
## Collection configuration
```

(We've started to replace 'class' with 'collection' in new materials. When we get to some critical mass of changes, we plan to backport the change across the rest of the docs.)

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+You can configure how the module will behave in each class through the [Weaviate schema](/developers/weaviate/configuration/schema-configuration.md).
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:59]: To configure how the module behaves in each collection, update the [Weaviate schema](/developers/weaviate/configuration/schema-configuration.md).

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+### API settings (OpenAI)
+
+#### Parameters
+
+|Parameter|Required|Default|Purpose|
+|:-|:-|:-|:-|
+|`model`|Optional|`jina-embeddings-v2-base-en`|A model name, e.g. `jina-embeddings-v2-small-en`.|
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:67]: Consider:
|`model`|No|

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+#### Example
+
+The following example configures the `Document` class by setting the vectorizer to `text2vec-jinaai` and the model to `jina-embeddings-v2-small-en`:
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:70]: Consider:
configures the `Document` collection

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+```json
+{
+  "classes": [
+    {
+      "class": "Document",
+      "description": "A class called document",
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:77]: Consider

"A collection called document" 

(Yes, this looks weird)

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+      // highlight-end
+    }
+  ]
+}
+```
+
+
+### Vectorization settings
+
+You can set vectorizer behavior using the `moduleConfig` section under each class and property:
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:94]: Consider:
section under each collection

File: developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+
+#### Class-level
+
+|Parameter|Default|Purpose|
+|:-|:-|:-|
+|`vectorizer`|| Use this module to vectorize the data.|
```
Comment (daveatweaviate) [inline developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md:100]: Consider

|`vectorizer`|-|

**Review by daveatweaviate — COMMENTED** [review]
@JoanFM  Thanks for your updates, I've left a few suggestions. Please let me know if you have any questions.

**Comment by daveatweaviate** [comment]
@JoanFM - on second thought, let me pull in your PR and update it here. Thanks again for the update!

**Comment by JoanFM** [comment]
> @JoanFM - on second thought, let me pull in your PR and update it here. Thanks again for the update!

Feel free to update, perfect

**Comment by daveatweaviate** [comment]
Closing in favor of [PR 1282](https://github.com/weaviate/weaviate-io/pull/1282/)

**Pull request closed without merging**